extensions/amp-story/1.0/bookend/components/cta-link.js

```diff
@@ -106,8 +112,15 @@ export class CtaLinkComponent {
       const refs = htmlRefs(el);
       refs['linkText'].textContent = currentLink['text'];
 
+      el[AMP_STORY_BOOKEND_COMPONENT_DATA] = {
+        position: data.position++,
```

Comment:
Is this how it behaves?

- 0 heading
- 1 small
- 2 small
- 3 link
  - 4 link
  - 5 link
  - 6 link
- 4 small

Reply:
Nope, like this:

* 0 heading
* 1 small
* 2 small
* 3 link 
* 4 link
* 5 link 
* 6 small 

Hence the reduction of `data.position` below in line 123. I think this is what would be easier to understand for publishers. WDYT?

Reply:
SG. I will keep the position as the index of the component (links sharing the same position).

P.S. if you were curious about the decrementing one time: since the increment was happening in the inner loop of `cta-link.js`, then again in the outer loop of `bookend-component.js` I needed to decrement one time to avoid skipping a position. This should be less confusing with the new logic.